DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Package/WebApiLibraryPackage.cs

```diff
@@ -10,6 +14,10 @@ public void RegisterServices(Container container)
         {
             container.Register<IGithubService, GithubService>(Lifestyle.Singleton);
             container.Register<IAuthService, AuthService>(Lifestyle.Scoped);
+            var assemblies = AppDomain.CurrentDomain.GetAssemblies().ToList();
+            container.Register(typeof (IValidator<>), assemblies, Lifestyle.Singleton);
+            container.RegisterConditional(typeof (IValidator<>), typeof (NullValidator<>), Lifestyle.Singleton,
```

Comment:
Yes. FIrst we search through our assembly. And then register all validators that implement IValidator<>.
The last line need to register NullValidator for all entities that haven't own validators. In other case we will have runtime error when try to work with such entity.

I think I should replace first 2 lines on this 
var assemblies = new[] { typeof(IValidator<>).Assembly };
container.RegisterCollection(typeof(IValidator<>), assemblies);
according to the best practice in SI docs.

https://simpleinjector.readthedocs.org/en/latest/advanced.html#batch-registration